### ISSUE TITLE
Diff view for sync conflict files?

### ISSUE THREAD

**Issue by dinesh-58** [orgzly-revived/orgzly-android-revived#292]
- I have searched for existing issues that may be the same as or related to mine.

Maybe a git diff-esque feature for viewing differences between a file and its sync-conflict counterpart to see what notes haven't been synced? Might be possible if I was using git to sync but I'm currently using Syncthing.

**Comment by gursuj** [comment]
For anyone seeing this, a workaround is to use emacs' [diff](https://www.gnu.org/software/emacs/manual/html_node/emacs/Comparing-Files.html) command.
```
M-x diff RET a.org RET a-sync-conflict.org RET
```

**Comment by alensiljak** [comment]
I would be categorically against adding such functionality into Orgzly. This is a task management app. There are specialized tools for pretty much anything else.
Sync, git, conflicts, and who knows what else, would be better off handled by a specialized app that focuses on that only.

**Comment by dinesh-58** [comment]
Understandable. Seems the syncthing devs are also not interested in developing such features ([1](https://github.com/syncthing/syncthing/issues/6332), [2](https://github.com/syncthing/syncthing/issues/9280)) and external tools are enough for this purpose.

**Comment by alensiljak** [comment]
Files are just not meant for that purpose, I guess.

**Comment by mlt** [comment]
I believe quick viewing the diff would be useful from user's perspective. Also, I presume, the "conflicting" changes are likely in different parts of a file. If we can snap a *base* version upon a successful sync, then produce a diff, and try to "patch" the remote, it would apply cleanly more often than not.

**Comment by gursuj** [comment]
Agreed. I recently looked for Android apps that could do text file diff but most seemed to have been removed from the play store and I can't find apks for them. Closest was [Total Commander](https://www.ghisler.ch/board/viewtopic.php?p=400862&sid=b7aa1e6471cc1fff445d3a9c2ff216c3#p400862). Probably possible with termux but that might be a hassle compared to GUI options within orgzly itself

**Comment by alensiljak** [comment]
I was using git for the repository and this functionality is then provided by the tooling.
